src/utils/url.ts

```diff
@@ -105,6 +105,22 @@ export const isValidUrl = (url: string): boolean => {
   }
 };
 
+// @debt possibly merge this and isValidUrl right above
+export const isStringAValidUrl = (urlString: string) => {
+  if (!urlString) return false;
+
+  try {
+    const url = new URL(urlString);
+
+    return VALID_URL_PROTOCOLS.includes(url.protocol);
+  } catch (e) {
+    if (e.name === "TypeError") {
```

Comment:
e here is of type `any` and it's better not to use `name` to check for the error type but use `instanceof`

Reply:
fixed d0f3e28